sormas-ui/src/main/java/de/symeda/sormas/ui/dashboard/DashboardDataProvider.java

```diff
@@ -150,18 +148,21 @@ private void refreshDataForSelectedDisease() {
 		if (getDashboardType() == DashboardType.CONTACTS || this.disease != null) {
 			// Cases
 			CaseCriteria caseCriteria = new CaseCriteria();
-			caseCriteria.region(region).district(district).disease(disease).newCaseDateBetween(fromDate, toDate, newCaseDateType);
+			caseCriteria.region(region).district(district).disease(disease).newCaseDateType(newCaseDateType).newCaseDateBetween(fromDate, toDate);
 			setCases(FacadeProvider.getCaseFacade().getCasesForDashboard(caseCriteria));
-			setLastReportedDistrict(FacadeProvider.getCaseFacade().getLastReportedDistrictName(caseCriteria, true, true));
+			setLastReportedDistrict(FacadeProvider.getCaseFacade().getLastReportedDistrictName(caseCriteria, true));
 
-			caseCriteria.newCaseDateBetween(previousFromDate, previousToDate, newCaseDateType);
+			caseCriteria.newCaseDateBetween(previousFromDate, previousToDate);
 			setPreviousCases(FacadeProvider.getCaseFacade().getCasesForDashboard(caseCriteria));
 
 			if (getDashboardType() != DashboardType.CONTACTS) {
 				if (getCases().size() > 0) {
-					setTestResultCountByResultType(
-						FacadeProvider.getSampleFacade()
-							.getNewTestResultCountByResultType(getCases().stream().map(c -> c.getId()).collect(Collectors.toList())));
+					List<Long> list = new ArrayList<>();
```

Comment:
What's the reason for turning this into a for loop?

Reply:
It was a refactoring suggested by IDE: Lambda can be replaced with method reference. Should I revert it?